gemfury.gemspec

```
@@ -17,7 +17,7 @@ Gem::Specification.new do |s|
                         Dir.glob("lib/**/*")
 
   s.add_dependency    "multi_json", "~> 1.10"
-  s.add_dependency    "thor", ">= 0.14.0", "< 1.0.0.pre"
+  s.add_dependency    "thor", ">= 0.14.0"
```

I didn't see a versioning policy, so I'm not sure if it's SemVer or else. Probably should cap the range until more clarity on their API stability. 

```suggestion
  s.add_dependency    "thor", ">= 0.14.0", "< 1.1.0"
```